ALICE3/Core/DelphesO2TrackSmearer.cxx

```diff
@@ -224,13 +224,13 @@ bool TrackSmearer::smearTrack(O2Track& o2track, int pdg, float nch)
 }
 
 /*****************************************************************/
-// relative uncertainty on pt
+// absolute uncertainty on pt
```

Comment:
Hi @NNicassio99 is this intended, it would break also the other users

Reply:
Hi @njacazio thanks for also taking a look at the code.
When I implemented this, it was thought as a starting point for the whole pid in nsigmas.
In case Nicola needs these changes, I think it would be Ok.
I suggest checking that the uncertainties are all returned in absolute values.

Reply:
Hi @hscheid in principle the smearer should not be changed as a core. What you can do is to add pieces if you think that something is missing

Reply:
If I am not wrong, this is a custom getter that is not used in the smearer itself. The smearTrack methods use the LUT entries directly. Maybe another approach could be to implement a second method that returns the absolute uncertainty values.

Reply:
Yes, that would be good, or get the relative uncertainty and scale it by pT or so